src/app/pages/BridgeWithdrawPage/components/AmountSelector/index.tsx

```diff
@@ -155,93 +155,97 @@ export const AmountSelector: React.FC = () => {
+            {t(trans.dailyBridgeLimits)}
+          </div>
+          <Table className="tw-w-full">
+            <tbody className="tw-text-left tw-text-sm tw-font-medium">
+              <tr>
+                <td> {t(trans.minAmount)}:</td>
+                <td className="tw-text-right">
+                  <LoadableValue
+                    value={`${toNumberFormat(
+                      currentAsset.fromWei(limits.returnData.getMinPerToken),
+                      currentAsset.minDecimals,
+                    )} ${currentAsset.symbol}`}
+                    loading={limitsLoading}
+                  />
+                </td>
+              </tr>
+              <tr>
+                <td> {t(trans.maxAmount)}:</td>
+                <td className="tw-text-right">
+                  <LoadableValue
+                    value={`${toNumberFormat(
+                      fromWei(limits.returnData.getMaxTokensAllowed),
+                      currentAsset.minDecimals,
+                    )} ${currentAsset.symbol}`}
+                    loading={limitsLoading}
+                  />
+                </td>
+              </tr>
               <tr>
-                <td> {t(trans.aggregatorBalance)}</td>
-                <td>
+                <td> {t(trans.dailyLimit)}:</td>
+                <td className="tw-text-right">
                   <LoadableValue
                     value={`${toNumberFormat(
-                      currentAsset.fromWei(bridgeBalance.value),
-                      withdrawAsset.minDecimals,
-                    )} ${withdrawAsset.symbol}`}
-                    loading={bridgeBalance.loading}
+                      fromWei(limits.returnData.dailyLimit),
+                      currentAsset.minDecimals,
+                    )} ${currentAsset.symbol}`}
+                    loading={limitsLoading}
                   />
                 </td>
               </tr>
-            )}
-          </tbody>
-        </Table>
+              <tr>
+                <td> {t(trans.dailyLimitSpent)}:</td>
+                <td className="tw-text-right">
+                  <LoadableValue
+                    value={`${toNumberFormat(
+                      fromWei(limits.returnData.spentToday),
+                      currentAsset.minDecimals,
+                    )} ${currentAsset.symbol}`}
+                    loading={limitsLoading}
+                  />
+                </td>
+              </tr>
+              <tr>
+                <td> {t(trans.fee)}:</td>
+                <td className="tw-text-right">
+                  <LoadableValue
+                    value={`${toNumberFormat(
+                      currentAsset.fromWei(limits.returnData.getFeePerToken),
+                      currentAsset.minDecimals,
+                    )} ${currentAsset.symbol}`}
+                    loading={limitsLoading}
+                  />
+                </td>
+              </tr>
+              {bridgeBalance.value !== false && (
```

Comment:
I don't get why the `bridgeBalance.value` is either `string` or `false`. Instead of `false`, assign `0` (or `null`).